contracts/BatchExchangeViewer.sol

```diff
@@ -28,7 +30,10 @@ contract BatchExchangeViewer {
      */
     function getOpenOrderBook(address[] memory tokenFilter) public view returns (bytes memory) {
         (bytes memory elements, , , ) = getOpenOrderBookPaginated(tokenFilter, address(0), 0, LARGE_PAGE_SIZE);
-        require(elements.length < LARGE_PAGE_SIZE * AUCTION_ELEMENT_WIDTH, "Orderbook too large, use paginated view functions");
+        require(
+            elements.length < uint256(LARGE_PAGE_SIZE) * INDEXED_AUCTION_ELEMENT_WIDTH,
```

Comment:
why was this casting not used before? Has something changed regarding the type or did adding 2 * 5000 make it overflow?

Reply:
This would have overflown in the past since LARGE_PAGE_SIZE is uint16 and thus bound to 65k. 5000 * 112 is 560k and thus modulos to ~318 elements rather than 5k